graphdatascience/aura_api.py

```diff
@@ -54,26 +55,19 @@ def fromJson(cls, json: dict[str, Any]) -> InstanceSpecificDetails:
 
 
 @dataclass(repr=True)
-class InstanceCreateDetails(InstanceDetails):
-    password: str
+class InstanceCreateDetails:
+    id: str
     username: str
+    password: str
     connection_url: str
```

Comment:
so we only look at the fields we actually need and not everything that should be returned?

Reply:
Exactly. This protects us from the flakiness of the `region`, `name`, and `type` fields, which we do not need.